plugins/BEdita/API/tests/TestCase/Controller/ObjectsControllerTest.php

```diff
@@ -65,20 +65,22 @@ public function testIndex()
                     'attributes' => [
                         'status' => 'on',
                         'uname' => 'first-user',
-                        'locked' => true,
-                        'created' => '2016-05-13T07:09:23+00:00',
-                        'modified' => '2016-05-13T07:09:23+00:00',
-                        'published' => null,
                         'title' => 'Mr. First User',
                         'description' => null,
                         'body' => null,
                         'extra' => null,
                         'lang' => 'eng',
-                        'created_by' => 1,
-                        'modified_by' => 1,
                         'publish_start' => null,
                         'publish_end' => null,
                     ],
+                    'meta' => [
+                        'locked' => true,
```

Comment:
Yes, I think so. Maybe some fields should be not visible in `attributes` unless you are `administrator` and if you aren't an admin you can just see it in `meta` (or not even in `meta` if is confusing)